drivers/disk/sdmmc_stm32.c

```diff
@@ -923,6 +933,163 @@ void stm32_sdmmc_get_card_csd(const struct device *dev, uint32_t csd[4])
 	memcpy(csd, &priv->hsd.CSD, sizeof(priv->hsd.CSD));
 }
 
+static int stm32_sdmmc_activate(const struct device *dev)
+{
+	pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
```

Comment:
Is this needed?

Mot of the sequence is already done in in `stm32_sdmmc_access_init()`. Could you factorize their common code?